Makefile

```diff
@@ -210,6 +194,7 @@ IDENTITY_ALIAS_LIST:=istioctl mixc mixs pilot-agent servicegraph sidecar-initial
 .PHONY: $(IDENTITY_ALIAS_LIST)
 $(foreach ITEM,$(IDENTITY_ALIAS_LIST),$(eval $(ITEM): ${ISTIO_BIN}/$(ITEM)))
 
+# SHRIRAM: Do we need these?
```

Comment:
Those three are only there because the aliases were pre-existing and I didn't feel like breaking existing users or asking around to see if anyone objected to me changing the names.  It's be simpler to delete the istio-ca, node-agent, and pilot targets and instead add "istio_ca node_agent pilot-discovery" to IDENTITY_ALIAS_LIST.  I'm guessing the biggest objection would come from people who don't want to type the longer "make pilot-discovery" instead of "make pilot".

Reply:
Or we can simply delete them - make should build all binaries, for development on individual component people can use the go tools directly ?

Reply:
`make pilot` or `make mixer` etc... could be nice for people who know they only change one or the other ?